### DISCUSSION TITLE
Rider's context actions are delaying down after referencing Metalama project

### DISCUSSION THREAD

**Discussion by mqsrr** [metalama/Metalama#254] [Q&A] [answered]
I've been working on a project that intercepts repository methods and implements cache logic using `Metalama.Framework 2024.0.6`. While the cache mechanism appears to be working well, I've noticed that when I reference a project to test interceptors, my context actions `Alt + Enter` slow down and require me to wait 2-3 seconds before they are enabled. I have not tested this behavior on other IDEAs.

![image](https://github.com/postsharp/Metalama/assets/82763271/b62d2d01-4ce3-4500-acf0-6987391e817d)

I tried removing all the attributes and fabrics, but nothing changed. The Metalama project does not have this issue, however the other projects have.
I'd appreciate any help on how to overcome this issue.

**Comment by svick** [comment]
Thanks for the report, I'm looking into this.

**Comment by prochan2** [comment, marked as answer]
This bug has been fixed in [Metalama 2024.1.1-preview](https://github.com/orgs/postsharp/discussions/256).
